lib/rules/no-unknown-property.js

```diff
@@ -205,11 +206,11 @@ function getStandardName(name, context) {
   }
   let i = -1;
   const names = getDOMPropertyNames(context);
-  const found = names.some((element, index) => {
-    i = index;
-    return element.toLowerCase() === name;
-  });
-  return found ? names[i] : null;
+  // Let's find a possible attribute match with a case-insensitive search.
+  const foundName = names.find(
+    (element, index) => element.toLowerCase() === name.toLowerCase()
+  );
+  return foundName === undefined ? null : foundName;
```

Comment:
any reason not to just `return foundName` here, and have the function return `String | undefined`?

Reply:
There was 2 reasons:
* initially I wanted to keep it working just like before, without changing too much code -- I ended up changing more code than I wanted :)
* we have this habit in our project that we use `null` as an explicite "no value" whereas `undefined` is accidental.

In this context I agree this isn't so useful, where `getStandardName` is used at only one place. I can change this.